### PR TITLE
Java 17 and update

### PR DESCRIPTION
**Pull request by angel-perez-itti**
Pull-request for issue https://github.com/slingr-stack/platform/issues/15600 created by @pasaperez-slingr 

## What it does?

Migrate the proxy-service to use java 17. 
Now the pom.xml is less dependent on the build options. It only takes care of the service specific code.

## How to test it?

You can test this service running the http-service and this service in parallel and try execute a webhook, this should arrive to http-service.

Test with any service in develop branch.

Remember to use this [checklist](https://docs.google.com/document/d/139rKhIsvpT3yBSB88BBZfy03fD_Sried0bVwIWVaHJk/edit?usp=sharing) for the review.

## Technical notes

There aren't technical notes

## Deployment notes

Create a new tag and build in each version.

### CODE REVIEW

**Review by agreggio-slingr — APPROVED** [review]

**Comment by pasaperez-work** [comment]
I will merge this service after merge https://github.com/slingr-stack/platform/pull/15624 and https://github.com/slingr-stack/platform-sdk-services/pull/20

**Review by agreggio-slingr — APPROVED** [review]

**Pull request merged**
